frontend/integration-tests/views/monitoring.view.ts

```diff
@@ -69,16 +69,36 @@ export const saveAsDefault = firstElementByTestID('save-as-default');
 
 export const defaultAlertmanagerYaml = Base64.encode(`"global":
   "resolve_timeout": "5m"
+"inhibit_rules":
```

Comment:
OK, as a follow on, but we should read the initial config from the cluster instead of hard-coding here. We do this for the OAuth scenario:

https://github.com/openshift/console/blob/master/frontend/integration-tests/tests/oauth.scenario.ts#L10-L20

Reply:
Ok, but I believe the initial alertmanager conifg gets overwritten after updating the alertmanager Secret.

Reply:
> Ok, but I believe the initial alertmanager conifg gets overwritten after updating the alertmanager Secret.

`beforeAll` will read it before any tests are run

Reply:
I believe once you update the alert-main Secret with an updated alertmanager.yaml it overwrites the initial alertmanager config.  I'm not sure there is a way to get what the original was.

Reply:
Right, that's why you read the initial config before updating the secret :)

Reply:
ah, read and save and restore on `beforeAll` -got it :-)